framework/python/src/common/testreport.py

```diff
@@ -23,7 +23,7 @@
 
 DATE_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'
 RESOURCES_DIR = 'resources/report'
-TESTS_FIRST_PAGE = 12
+TESTS_FIRST_PAGE = 9

```

Comment:
This should be dynamic depending on the size of the header.  Making it 9 leaves a lot of dead space with any manufacturer not having too long of a name problem.

Reply:
It has now been set to 11 which should be enough space for a device with a long name, without causing much white space for short names.